src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -53,6 +53,11 @@ def handle_exception(ex):  # pylint: disable=too-many-return-statements
             return 1
         if isinstance(ex, (CLIError, CloudError, AzureException)):
             logger.error(ex.args[0])
+            try:
```

Comment:
besides CloudError, AzureException, is this need for CLIError also?

Reply:
Yes. In this example, ex is an CLIError. ex.args[0] is a CloudError.